flickr_download/flick_download.py

```diff
@@ -300,6 +303,26 @@ def print_sets(username):
         print('{0} - {1}'.format(photo.id, photo.title))
 
 
+def get_cache(path):
+    if not os.path.exists(path):
+        cache = SimpleCache(max_entries=20000, timeout=3600)
```

Comment:
Complete nit, but I’m a huge fans of early returns. So if you just do a `return SimpleCache(...` here you can lose the `else` below and the indent. Makes it easier to read imo.